kafka/consumer_test.go

```diff
@@ -46,13 +51,38 @@ func TestConsumerAPIs(t *testing.T) {
 		t.Fatalf("%s", err)
 	}
 
+	closedConsumer, err := NewConsumer(&ConfigMap{
+		"group.id":                 "gotest",
+		"socket.timeout.ms":        10,
+		"session.timeout.ms":       10,
+		"enable.auto.offset.store": false, // permit StoreOffsets()
```

Comment:
Keep only the required config options, the ones for example 'enable.auto.offset.store' is not needed for this particular consumer since we don't deal with any offsets at all.

Reply:
We did use Store Offsets method on closedConsumer and nilConsumer.